ggshield/sca/filter.py

```diff
@@ -0,0 +1,60 @@
+import re
+from pathlib import Path
+from typing import List, Set
+
+from ggshield.scan import Scannable
+from ggshield.scan.file import get_files_from_paths
+
+
+# List of filepaths to ignore for SCA scans
+
+SCA_IGNORE_LIST = (
+    "__pycache__",
+    ".git",
+    ".hg",
+    ".svn",
+    ".tox",
+    ".venv",
+    "site-packages",
+    ".idea",
+    "node_modules",
+    ".mypy_cache",
+    ".pytest_cache",
+    ".hypothesis",
+)
+
+
+def get_all_files_from_sca_potential_paths(
```

Comment:
Why "potential" paths ?

Also the docstring is wrong - but I see this is coming from IaC

Reply:
Thanks for the docstring. 
About `potential`, I wanted to say that many of those files will not be SCA files, but indeed just `get_all_files_from_sca_paths` is better.